cEP-0020.md

```diff
@@ -0,0 +1,462 @@
+Up` to the issue.
+
+Similarly in case of the pull requests if the mentioned issue in the commit is opened
+by newcomer and doesn't have `three` number of `Thumbs Up`, it will not qualify to
+be tracked in this system.
+
+- Create Issue
+  - 20 points for type/bug
+  - 20 points for bear proposal
+  - 10 points for difficulty newcomer
+  - 20 points for difficulty low
+  - 30 points for difficulty medium
+- Create Pull Requests
+  - No points will be given until all the CI on the pull request is passing
+  - If CI passes and there are no Gitmate errors
+    - 20 points for bug fixes
+    - 20 points for writing a bear
+    - 10 points for difficulty newcomer
+    - 20 points for difficulty low
+    - 30 points for difficulty medium
+  - If CI passes and Gitmate errors are b/w 1 to 5
+    - 10 points for bug fixes
+    - 10 points for writing a bear
+    - 5 points for difficulty newcomer
+    - 10 points for difficulty low
+    - 15 points for difficulty medium
+  - If CI passes and Gitmate errors are b/w 5 to 10
+    - 5 points for bug fixes
+    - 5 points for writing a bear
+    - 3 points for difficulty newcomer
+    - 5 points for difficulty low
+    - 10 points for difficulty medium
+  - If CI passes and Gitmate errors are b/w 10 and beyond
+    - No points given
+  - If the pull request gets merged
+    - 5 points for merging any pull request
+  - If the pull request is closed without merged
+    - All the points given for that pull request will be reset
+- Activities related to introducing `.coafile`
+  - 10 points
+- Activities related to setting up CIs to the fork
+  - 10 points
+
+### Levels
+
+To increase a user's competitive instinct, we can implement a series of
+levels that confer rank as newcomers become more active. A good and
+straightforward way to award levels in a new gamified design is to base them
+on point thresholds. As users earn points, they move up an incremental
+series of levels. To infuse the level with meaning i.e.: a ranking within the
+system, the levels should be named in a way that indicates status.
+
+But the levels wouldn't be enough motivation to go until newcomers get
+some special privileges after completing each level:
+
+- Level-I:
+  - Name: Fresher
+  - Points_required = 0
+  - Privileges:
+    - No privileges for freshers
+- Level-II:
+  - Name: Newbie
+  - Points_required = 5
+  - Privileges:
+    - Get assigned to newcomers issue
+- Level-III:
+  - Name: Beginner
+  - Points_required = 15
+  - Privileges:
+    - Get assigned to a difficulty low issue
+- Level-V:
+  - Name: Intermediate
+  - Points_required = 30
+  - Privileges:
+    - Mark your own PR as wip
+- Level-VI:
+  - Name: Cool
+  - Points_required = 50
+  - Privileges:
+    - Mark a PR by another newcomer as wip
+- Level-VII:
+  - Name: Awesome
+  - Points_required = 100
+  - Privileges
+    - Get assigned to difficulty medium issue
+- Level-VIII
+  - Name: Cool and Awesome
```

Comment:
At the risk of this level name being confusing due to the previous two, maybe something like "Legend" or "Master"? Not sure if those would go before or after "Expert", though.